src/app/pages/send/send-crypto-asset-form/_hooks/use-stacks-recipient-bns-name.tsx

```diff
@@ -0,0 +1,32 @@
+import { useCallback, useState } from 'react';
+
+import { useField } from 'formik';
+
+import { useStacksClientUnanchored } from '@app/store/common/api-clients.hooks';
+
+export function useStacksRecipientBnsName() {
```

Comment:
Some comment for this one. What do you think about the architectural rule where _all_ chain-specific code lives under it's designated folder?

Reply:
Same response.